python/.pylintrc

```diff
@@ -0,0 +1,574 @@
+[MASTER]
```

Comment:
Question: where does this config file come from? (I'm in no way a Python guru - so bear with me ;) )

Reply:
no problems :). It is generated through `pylint` cli via this command `pylint --generate-rcfile > .pylintrc` and then from there, the config file can be modified as needed

Reply:
So it's the default for now, right?

Reply:
Nope, I have disabled some features like `consider-using-f-string`, `too-few-public-methods`  .. etc as I think they are not necessary needed for now and changed properties like `max-line-length`